lib/idris-controller.coffee

```diff
@@ -46,6 +47,9 @@ class IdrisController
       @messages.attach()
       @messages.hide()
 
+  stopCompiler: =>
+    @model?.stop()
```

Comment:
I made the message less dire. It looks like this now if you stop the compiler manually:
![stop_compiler](https://cloud.githubusercontent.com/assets/471043/9909481/bccaa1aa-5c98-11e5-8ad6-83127b84cb1f.PNG)